oneview/data_source_server_profile.go

```diff
@@ -912,8 +912,12 @@ func dataSourceServerProfile() *schema.Resource {
 							Elem: &schema.Resource{
 								Schema: map[string]*schema.Schema{
 									"initial_scope_uris": {
-										Type:     schema.TypeString,
-										Optional: true,
+										Computed: true,
```

Comment:
since you have made initial_scope_uri to Computed, 
If user wants to remove a scope uri, then how will he do that?